applications/nrf5340_audio/src/bluetooth/bt_management/scanning/bt_mgmt_scan_for_conn.c

```diff
@@ -182,7 +244,11 @@ static void scan_recv_cb(const struct bt_le_scan_recv_info *info, struct net_buf
 	case BT_GAP_ADV_TYPE_SCAN_RSP:
 		/* Note: May lead to connection creation */
 		if (bonded_num < CONFIG_BT_MAX_PAIRED) {
-			bt_data_parse(ad, device_name_check, (void *)info->addr);
+			if (sirk_local == NULL) {
```

Comment:
What is meant by `sirk_local` on the unicast client side?

Reply:
It is just a way to temporary store the sirk of the set we are currently looking for. Since we are using the `bt_conn_foreach()` we need to store the pointer locally. We could use a struct and then pack the num_filled and sirk in there, it just seemed easier this way.